src/RequestHandle.cpp

```diff
@@ -215,9 +216,28 @@ auto RequestHandle::SetVersion(
     }
 }
 
-auto RequestHandle::SetMaxBytes(size_t max_bytes) -> void
+auto RequestHandle::SetMaxDownloadBytes(ssize_t max_download_bytes) -> void
```

Comment:
So the init() function is duplicating this work, why not just have init() call this function when its setting the max_download_bytes

Reply:
Good point! Added call to init() and removed duplicate code.